server/src/test/java/org/opensearch/cluster/ClusterStateTests.java

```diff
@@ -84,7 +86,7 @@ public void testSupersedes() {
         final DiscoveryNode node1 = new DiscoveryNode("node1", buildNewFakeTransportAddress(), emptyMap(), emptySet(), version);
         final DiscoveryNode node2 = new DiscoveryNode("node2", buildNewFakeTransportAddress(), emptyMap(), emptySet(), version);
         final DiscoveryNodes nodes = DiscoveryNodes.builder().add(node1).add(node2).build();
-        ClusterName name = ClusterName.CLUSTER_NAME_SETTING.getDefault(Settings.EMPTY);
+        ClusterName name = CLUSTER_NAME_SETTING.getDefault(Settings.EMPTY);
```

Comment:
nit - unrelated here.

Reply:
Yup, it is unrelated. This was change automatically, as I introduced new tests which refered CLUSTER_NAME_SETTING